include/fastrtps/utils/TimedConditionVariable.hpp

```diff
@@ -116,15 +116,11 @@ class TimedConditionVariable
                 const std::chrono::steady_clock::time_point& max_blocking_time,
                 std::function<bool()> predicate)
         {
+            auto secs = std::chrono::time_point_cast<std::chrono::seconds>(max_blocking_time);
```

Comment:
Could you bring the `wait_until` with predicate to master? It will be useful to avoid the error of leaving the old version when we bring the forward port of the reimplementation of time events.

Reply:
> Could you bring the `wait_until` with predicate to master?

I think you refer to the one __*without*__ predicate

Reply:
done